application/src/main/resources/thingsboard.yml

```diff
@@ -978,6 +978,22 @@ transport:
     enabled: "${TB_TRANSPORT_STATS_ENABLED:true}"
     print-interval-ms: "${TB_TRANSPORT_STATS_PRINT_INTERVAL_MS:60000}"
 
+# Device connectivity properties to publish telemetry
```

Comment:
Plz add ability to enable/disable per each protocol
security protocols by default should be disabled